tcms/core/tests/test_middleware.py

```diff
@@ -10,9 +11,19 @@
 class TestCheckUnappliedMigrationsMiddleware(test.TransactionTestCase):
     def test_unapplied_migrations(self):
         call_command('migrate', 'bugs', 'zero', verbosity=2, interactive=False)
-        unapplied_migration_message = 'unapplied migration(s). See '\
-            '<a href="https://kiwitcms.readthedocs.io/en/latest/'\
-            'installing_docker.html#initial-configuration-of-running-'\
+        unapplied_migration_message = 'unapplied migration(s). See ' \
+            '<a href="https://kiwitcms.readthedocs.io/en/latest/' \
+            'installing_docker.html#initial-configuration-of-running-' \
```

Comment:
These changes are unrelated and cause a conflict. 

In addition we've started using black to format the code. All existing code has been reformatted and a CI check has been added. Rebase to latest master, resolve conflicts and update where necessary.